app/views/event_attendances/index.html.erb

```diff
@@ -1,91 +1,123 @@
 <h3 class="pre-title"><%= @event.name %></h3>
 <h1><%=h @audience.to_s %> Attendance</h1>
-	
+    
 <table class="attendees">
-	<thead>
-		<tr>
-			<th>Attendee</th>
-			<%= content_tag(:th, "High School") if @audience.to_s.downcase == "participant" || @audience.to_s.downcase == "student" %>
-			<th>Program(s)</th>
-			<th>Phone</th>
-			<th>Shift</th>
-			<%= "<th class='centered'>Trained?</th>" if @event.training_required?(@audience.to_s) %>
-			<th class="centered">RSVP'd?
-				<%= link_to "E-mail",
-				 				"mailto:#{@event.attendees.rsvpd(@audience).collect{|a| a.email rescue nil }.compact.join(",")}",
-								:class => 'email button',
-								:id => 'attending_email_link' %>
-				</th>
-			<th class="centered">Attended?
-				<%= link_to "E-mail",
-				 				"mailto:#{@event.attendees.attended(@audience).collect{|a| a.email rescue nil }.compact.join(",")}",
-								:class => 'email button',
-								:id => 'attended_email_link' %>
-				</th>
-			<th>Functions</th>
-		</tr>
-	</thead>
-
-	<tbody>
-		<%= render :partial => 'event_attendance', :collection => @event.attendees.audience(@audience).select{|a| a.rsvp? || a.attended? || a.admin? }, :locals => { :audience => @audience } %>
-
-		<%- if @event.attendees.empty? -%>
-			<tr class="empty">
-				<td colspan=10>No event attendees yet.</td>
-			</tr>
-		<% end -%>
-	</tbody>
+    <thead>
```

Comment:
Watch what your editor is using for tab spacing, these were all tabs originally. @mattharris5 we should probably come up with a style guide to differentiate spacing between views and models, and to standardize method/variable names